app/models/outgoing_payment.rb

```diff
@@ -0,0 +1,5 @@
+class OutgoingPayment < ApplicationRecord
+  belongs_to :crime_application
+
+  store_accessor :metadata, :details
```

Comment:
Should this be added here, rather than in the form?

Reply:
In here, because ultimately this Model determines how data is saved in the database, the form object is more of a wrapper. It's russian dolls all the way down, with the Model being the tiny piece in the middle!